modules/serverless-creating-broker-annotation.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-using-brokers.adoc
+
+[id="serverless-creating-broker-annotation_{context}"]
+= Creating a broker by annotating a trigger
+
+[IMPORTANT]
+====
+If you create a broker by using the `eventing.knative.dev/injection=enabled` annotation, you cannot delete this broker without cluster administrator permissions.
+If you delete the broker without having a cluster administrator remove this annotation first, the broker will be created again after deletion.
+====
+
+.Prerequisites
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {product-title} cluster.
+
+.Procedure
+* Create a `Trigger` YAML object that has the `eventing.knative.dev/injection=enabled` annotation:
++
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1beta1
```

Comment:
is this for 1.11 ? 

if so, we have 0.17 - which comes w/ v1 types, see upstream doc:
https://github.com/knative/docs/tree/release-0.17/docs/eventing/triggers